rllib/agents/dqn/dqn.py

```diff
@@ -249,11 +263,22 @@ def update_prio(item):
                 _fake_gpus=config["_fake_gpus"],
             )
 
+        if (
```

Comment:
Ah, got it. Answers my question from above.

Is there a more elegant way of doing this? Like could we simply check, whether `self.local_replay_buffer` is a sub-class of a prio-replay one? That would ensure that the method: `update_priorities` is available.